app/components/settings/contact-info-section.js

```diff
@@ -30,8 +30,9 @@ export default Component.extend(FormMixin, {
           identifier : 'phone',
           rules      : [
             {
-              type   : 'empty',
-              prompt : this.get('l10n').t('Please enter your phone')
+              type   : 'regExp',
```

Comment:
@Anupam-dagar , I don't think you can remove the empty condition. 
Shouldn't there be two rules like in the email 1 for empty and the other for checking the valid regexp?

Also regexp isn't directly added in the component file.
Check how it's done for validating websites, twitter, facebook handles in session-speakers form
https://github.com/fossasia/open-event-frontend/blob/development/app/components/forms/session-speaker-form.js

Reply:
I agree with the 2 rules, I was implementing them but did a syntax mistake. Now with reference to the correct file, I got the correct syntax.